src/panel/direction/RouteSummary.jsx

```diff
@@ -37,6 +37,7 @@ export default class RouteSummary extends React.Component {
     return <Flex
       className="itinerary_leg_summary"
       justifyContent="space-between"
+      alignItems="flex-end"
```

Comment:
This results in a proptype console warning, as this value isn't explicitely declared yet in our `<Flex>` component.